pkg/controller/command/sdscomm/sdscomm.go

```diff
@@ -34,12 +38,21 @@ type DIDDocData struct {
 	Name     string          `json:"name,omitempty"`
 }
 
-func New(sdsServerURL, agentUsername string) *SDSComm {
+func New(sdsServerURL, agentUsername string) (*SDSComm, error) {
+	logger.Infof("!!!DEREK about to check if the provided SDS URL is blank.")
```

Comment:
`DEREK` ???

Reply:
Oops... fixed :)

Reply:
(just removed the log line - didn't mean to commit that...)